src/include/daos_srv/container.h

```diff
@@ -66,20 +66,11 @@ struct ds_cont_child {
 	ABT_cond		 sc_scrub_cond;
 	ABT_cond		 sc_rebuild_cond;
 	ABT_cond		 sc_fini_cond;
-	uint32_t		 sc_dtx_resyncing:1,
-				 sc_dtx_reindex:1,
-				 sc_dtx_reindex_abort:1,
-				 sc_dtx_delay_reset:1,
-				 sc_dtx_registered:1,
-				 sc_props_fetched:1,
-				 sc_stopping:1,
-				 sc_destroying:1,
-				 sc_vos_agg_active:1,
-				 sc_ec_agg_active:1,
-				 /* flag of CONT_CAPA_READ_DATA/_WRITE_DATA disabled */
-				 sc_rw_disabled:1,
-				 sc_scrubbing:1,
-				 sc_rebuilding:1;
+	uint32_t                 sc_dtx_resyncing : 1, sc_dtx_reindex : 1, sc_dtx_reindex_abort : 1,
+	    sc_dtx_delay_reset : 1, sc_dtx_registered : 1, sc_props_fetched : 1, sc_stopping : 1,
+	    sc_destroying : 1, sc_vos_agg_active : 1, sc_ec_agg_active : 1,
+	    /* flag of CONT_CAPA_READ_DATA/_WRITE_DATA disabled */
+	    sc_rw_disabled : 1, sc_scrubbing : 1, sc_rebuilding : 1, sc_open_initializing : 1;
```

Comment:
Is this the recommended coding style?

Reply:
this is actually only suppose to change 1 line, not sure why it messed up like this, which might due to git-hook sth.

Reply:
This might be messed up by clang-format hooks.

Reply:
verified (though we do this in other places in DAOS)

```
#include <stdint.h>
#include <stdio.h>

struct foo {
	uint64_t foo;
	uint32_t bar:1;
	uint32_t fub:1;
	uint32_t other;
};

int main(int argc, char ** argv)
{
	printf("%zu\n", sizeof(struct foo));
}
```

This prints 16 which means bar and fub are placed together.

Reply:
in fact, you can make this change without any formatting and clang-format will fix it and make it look nice